src/test/java/hudson/plugin/versioncolumn/JVMVersionComparatorTest.java

```diff
@@ -81,34 +87,34 @@ private static Object[] parametersForIncompatibleBytecodeLevel() {
     @MethodSource("parametersForCompatibleBytecodeLevel")
     public void compatibleBytecodeLevel(String agentVMVersion, final int masterBytecodeMajorVersion) {
         assertTrue(new JVMVersionComparator("whatever", agentVMVersion,
-                                            JVMVersionComparator.ComparisonMode.RUNTIME_GREATER_OR_EQUAL_MASTER_BYTECODE,
-                                            new JVMVersionComparator.MasterBytecodeMajorVersionNumberGetter() {
-                                                @Override
-                                                public int get() {
-                                                    return masterBytecodeMajorVersion;
-                                                }
-                                            }).isCompatible());
+                JVMVersionComparator.ComparisonMode.RUNTIME_GREATER_OR_EQUAL_MASTER_BYTECODE,
+                new JVMVersionComparator.MasterBytecodeMajorVersionNumberGetter() {
+                    @Override
+                    public int get() {
+                        return masterBytecodeMajorVersion;
+                    }
+                }).isCompatible());
```

Comment:
Gratuitous formatting changes; suggest reverting.

Reply:
Reverted in 361e5c0eb97b08e16b1d6d05b8e1b3c39f30fdb8.  Thanks!